apps/book/schema.py

```diff
@@ -53,7 +54,9 @@ class Meta:
 
 
 class BookType(DjangoObjectType):
-    quantity_in_cart = graphene.Int(required=True)
+    cart_details = graphene.Field(CartItemType)
+    # Wishlist id may be None
```

Comment:
This is self-explanatory, no need to specify.

Reply:
@thenav56  Removed this